### PR TITLE
Changelog for version 64

### PR DESCRIPTION
**Pull request by guy-har**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## v0.64.0 - 2022-04-29
+
+- Fix bug in merge (#3270)
```
Comment (nopcoder) [inline CHANGELOG.md:5]: and you add that this is related to the case of no changes on source

**Review by nopcoder — APPROVED** [review]
Approved, just update the first bug description

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## v0.64.0 - 2022-04-29
+
+- Fix bug in merge - merge with no changes resolves by creating empty commit (with no data) on destinations head  (#3270)
+- Fix broken content-based diff for changed objects in compare view (#3275)
+- Bump metadata client version to 0.1.7-RC.0 (#3277)
```
Comment (itaiad200) [inline CHANGELOG.md:7]: GC isn't released together with a lakeFS release, right? In that case I think we should avoid adding it to the lakeFS changelog (But there's really no need to fix it now).

Comment (arielshaqed) [reply CHANGELOG.md:7]: Yup, my mistake there.  OTOH it _does_ sort-of state that we are now testing after cutting this release with lakeFS (we always test HEAD, but obviously a release is a nice step to indicate along the way...).

**Pull request merged**